Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+@Library('mdblp-library') _
+
+pipeline {
+    agent {
+        label 'blip'
+    }
+    environment {
+        node_version='10.15'
```

Comment:
Since we are changing our build method, and now deploying on CloudFront, I will have use the latest lts nodejs version.
Or at least the latest LTS on a specific branch (v12 for example)